docs/Makefile.am

```diff
@@ -112,6 +111,8 @@ CD2_ = $(CD2_0)
 
 SUFFIXES = .1 .md
 
+all: $(man_MANS) $(MK_CA_DOCS)
```

Comment:
`all: $(man_MANS)` is unnecessary; automake does this automatically.

Reply:
But it does not depend on/build `$(MK_CA_DOCS)` automatically without this

Reply:
Correct. `all: $(MK_CA_DOCS)` is still needed.

Reply:
:+1: